describe.go

```diff
@@ -13,6 +14,13 @@ import (
 
 var defaultSpecName = "ISO 8583"
 
+var panFilter = regexp.MustCompile(`^\d(\d*)\d{4}$`)
+
+const (
+	bitmapIndex = 1
+	panIndex    = 2
+)
+
 func Describe(message *Message, w io.Writer) error {
```

Comment:
FYI human-readable output typically isn't protected from breaking changes, so we're free to alter this as needed. Hiding PII is very important. 

We offer multiple machine readable formats which we promise to not break.